apps/coscrad-frontend/src/components/tags/tag-detail.presenter.tsx

```diff
@@ -9,51 +9,25 @@ import { thumbnailCategorizableDetailPresenterFactory } from '../resources/facto
  * TODO[https://www.pivotaltracker.com/story/show/183618856]
  * We need to expose Tag commands through Tag queries.
  */
-export const TagDetailPresenter = ({ id, label, members }: ITagViewModel): JSX.Element => {
-    const allResourceTypesWithRepeats = members.flatMap(({ type }) => type);
-
-    const uniqueResourceTypesWithThisTag = [...new Set(allResourceTypesWithRepeats)];
-
-    const initialEmptyReosurceTypesAndSelectedIds = uniqueResourceTypesWithThisTag.reduce(
-        (acc, resourceType) => ({
-            ...acc,
-            [resourceType]: [],
-        }),
-        {}
-    );
-
-    // A Map might improve readability here
-    /**
-     * TODO Let's update `CategorizablesOfMultipleTypesContainer` to take in `members`
-     */
-    const resourceTypesAndSelectedIds = members.reduce(
-        (acc, { type: resourceType, id }) => ({
-            ...acc,
-            [resourceType]: acc[resourceType].concat(id),
-        }),
-        initialEmptyReosurceTypesAndSelectedIds
-    );
-
-    return (
-        <div data-testid={id}>
-            <Card>
-                <CardHeader title={label} />
-                <Divider />
-                <CardContent>
-                    <CategorizablesOfMultipleTypeContainer
-                        categorizableTypeAndIds={resourceTypesAndSelectedIds}
-                        detailPresenterFactory={thumbnailCategorizableDetailPresenterFactory}
-                        heading="Tagged Resources"
-                    />
-                </CardContent>
-                <CardActionArea>
-                    <CardActions>
-                        <Link to={id}>
-                            <LinkIcon />
-                        </Link>
-                    </CardActions>
-                </CardActionArea>
-            </Card>
-        </div>
-    );
-};
+export const TagDetailPresenter = ({ id, label, members }: ITagViewModel): JSX.Element => (
+    <div data-testid={id}>
+        <Card>
+            <CardHeader title={label} />
+            <Divider />
+            <CardContent>
+                <CategorizablesOfMultipleTypeContainer
+                    members={members}
+                    detailPresenterFactory={thumbnailCategorizableDetailPresenterFactory}
+                    heading="Tagged Resources"
+                />
+            </CardContent>
+            <CardActionArea>
```

Comment:
This may be the same navigation issue as with the all resources PR?  Could these cards just be the link themselves?

Reply:
Good point. I'm actually not sure why there is even a link here? This is **already** the detail presenter.